.github/scripts/get_system_info.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+#  Copyright (C) 2023 Intel Corporation
+#  Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See LICENSE.TXT
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# get-system-info.sh - Script for printing system info
+
+function system_info {
+	echo "********** system_info **********"
+	cat /etc/os-release | grep -oP "PRETTY_NAME=\K.*"
+	uname -r
```

Comment:
it seems this is not needed, since you're printing `/proc/version` below

Reply:
okey